lib/views/push-pull-view.js

```diff
@@ -215,14 +211,14 @@ export default class PushPullView extends React.Component {
               )}
               <span className={getIconClass(tileState.icon, tileState.iconAnimation)} />
               {tileState.text}
-            </Tag>
+            </span>
             <Tooltip
               key="tooltip"
               manager={this.props.tooltipManager}
               target={this.refTileNode}
               title={`<div style="text-align: left; line-height: 1.2em;">${tileState.tooltip}</div>`}
-              showDelay={200}
-              hideDelay={100}
+              showDelay={atom.tooltips.hoverDefaults.delay.show}
```

Comment:
it makes me nervous using dot notation to access deeply nested object properties, without existence checks.  However I think this is an atom api that's used in a lot of places, so maybe it's safe to assume this is ok.

Reply:
right right, I meant api as in interface/contract, that `atom.tooltips.whatever` would continue to be available.  Not api as in REST api. 😄